.claude/CLAUDE.md

```
@@ -23,3 +23,33 @@
 ### 出力先
 - 大規模タスク: `<project>/docs/specs/<task-name>-spec.md`
 - 小規模タスク: Plan Mode で使用される `.claude/plans/` 内のファイル
+
+## タスク管理ツールの積極活用
+
+作業の進捗を可視化し、複合的なタスクを構造的に管理するため、TaskCreate / TaskList / TaskUpdate を積極的に活用してください。
+
+### 自動実行トリガー
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)